src/report/main.ml

```diff
@@ -256,6 +256,16 @@ let coveralls =
 
 
 
+let merge =
+  let call_with_labels
+      to_file coverage_files coverage_paths expect do_not_expect =
+    Merge.output
+      ~to_file ~coverage_files ~coverage_paths ~expect ~do_not_expect
+  in
+  Term.(const set_verbose $ verbose $ const call_with_labels $ to_file
+    $ coverage_files 1 $ coverage_paths $ expect $ do_not_expect),
```

Comment:
Do you have a need for `--expect` and `--do-not-expect` in `bisect-ppx-report merge`? I suspect these arguments are only useful when generating a "true" final report. If that's right, I suggest to drop them from the command. You can provide `[]` where `~expect` and `~do_not_expect` are needed internally.

Reply:
I do not need them, I removed them in a287e8d7819c23b4b5a3e581f5be3929e005e805